ui/src/pages/services/index.tsx

```diff
@@ -9,20 +9,189 @@ export default function ServicesPage() {
       <SEO title="Services" description="" />
       <Section isHero theme="funky-5">
         <div className="grid">
-          <div>
-            <h1 className="margin-top-0">Services</h1>
+          <div className="mobile-reverse-item">
+            <h1 className="margin-top-0">Services Designed to Meet Your Modern Needs</h1>
             <p>
-              Stop settling for websites that sell your brand short. Here at Pure Func, we build sites that are just as
-              modern and innovative as your company, and we specialize in crafting gorgeous user interfaces that never
-              sacrifice performance. The bottom line? If you want a website that's exceptional in both form and
-              function, you're in the right place.
+              From ecommerce to web apps, Pure Func offers a wide variety of services to choose from. Whether you’re
+              looking to upgrade your old site or start a custom project from scratch, our extensive menu of modern,
+              relevant services will fulfill all of your web development needs.
             </p>
             <Link to="/contact" className="cta">
               Contact Us
             </Link>
           </div>
         </div>
       </Section>
+      <Section>
+        <div className="grid">
+          <div>
+            <h1 className="margin-top-0">Custom Websites</h1>
+            <p>
+              Bold. Smooth. Modern. Showcase the best of your work, products, and services with a custom user interface
+              designed to flatter your brand. Our static sites can be as simple or complex as you need, with the option
+              to add in e-commerce with Shopify or manage content through your favorite CMS, including Wordpress,
+              Contentful, Sanity, and more. Interested in a stunning static website but need it right away? Check out
+              our semi-custom development services (link) to get going right out of the box. See our custom website work
+              in action for Tune & Flying Home Studio (link).
```

Comment:
need an anchor tag on this with an href of https://flyinghomestudio.com/   also need to add a target of _blank and nofollow tags as well (see homepage for another example)